src/index.js

```diff
@@ -502,7 +497,12 @@ function createConfig(options, entry, format, writeMeta) {
 						extract:
 							!!writeMeta &&
 							options.css !== 'inline' &&
-							options.output.replace(EXTENSION, '.css'),
+							(options.multipleEntries
+								? resolve(
+										dirname(options.output),
+										`${basename(entry.replace(EXTENSION, ''))}.css`,
+								  )
+								: options.output.replace(EXTENSION, `.css`)),
```

Comment:
```suggestion
							absMain.replace(EXTENSION, '.css'),
```

We should be able to use `absMain` here instead to vastly simplify things. Should've been using it (rather than the `options.output.replace...`) from the start, but that's my fault.

Reply:
👌，I have already made changes to this